lib/property.rb

```diff
@@ -14,26 +14,45 @@ def initialize(id:, name:, description:, price:)
     @price = price
   end
   
-    def self.all
-        if ENV['ENVIRONMENT'] == 'test'
-            connection = PG.connect(dbname: 'makersbnb_test')
-        else
-            connection = PG.connect(dbname: 'makersbnb')
-        end
-        properties = connection.exec('SELECT * from properties;')
-        properties.map do |property| 
-          Property.new(
-            id: property['id'], 
-            name: property['name'], 
-            description: property['description'],
-            price: property['price'].to_i 
-          )
-        end
+  def self.all
+    if ENV['ENVIRONMENT'] == 'test'
+        connection = PG.connect(dbname: 'makersbnb_test')
+    else
+        connection = PG.connect(dbname: 'makersbnb')
     end
+    properties = connection.exec('SELECT * from properties;')
```

Comment:
get rid of if block
properties = DatabaseConnection.query('SELECT * from properties;')
**(properties/ results)